examples/NDBC_data.ipynb

```diff
@@ -0,0 +1,194 @@
+    "logging.basicConfig(\n",
+    "    level=20,\n",
+    "    style=\"{\",\n",
+    "    format=\"{asctime:s}; {levelname:8s}; {threadName:23s}; {name:<25s} {lineno:5d}; {message:s}\",\n",
+    ")\n",
+    "\n",
+    "logging.getLogger(\"urllib3\").setLevel(30)\n",
+    "logging.getLogger(\"parso\").setLevel(30)\n",
+    "\n",
+    "logger = logging.getLogger(__name__)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Retrieve all NDBC stations"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ndbc_stations = get_ndbc_stations()\n",
+    "ndbc_stations"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Visualize the location of the stations on a map"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create the plot\n",
+    "def plot_map(data, title):\n",
+    "\n",
+    "    # Plot the world map\n",
+    "    world_plot = data.hvplot(geo=True, tiles=True, hover_cols=[\"Station\", \"location\"])\n",
+    "    return world_plot.opts(width=800, height=500)\n",
+    "\n",
+    "\n",
+    "plot_map(ndbc_stations, 'NDBC Stations Worldwide')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Get the stations from a specific region and vizulaise it in a map"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from shapely.geometry import box\n",
+    "# Define a rectangular region\n",
+    "region = box(-85, 25, -65, 45)  # Longitude range, Latitude range\n",
+    "\n",
+    "# Get stations within the region\n",
+    "east_coast_stations = get_ndbc_stations(region=region)\n",
+    "plot_map(east_coast_stations, 'NDBC Stations on the East Coast of the US')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Not all stations provide the same set of measurements, the following measurements are listed and which type of stations support them:\n",
+    "\n",
+    "```markdown\n",
+    "\n",
+    "| Package Name          | C-MAN  | Moored |\n",
+    "|-----------------------|--------|--------|\n",
+    "| adcp                  | [ ]    | [ ]    |\n",
+    "| cwind                 | [X]    | [ ]    |\n",
+    "| ocean                 | [ ]    | [ ]    |\n",
+    "| spec                  | [ ]    | [ ]    |\n",
+    "| stdmet                | [X]    | [ ]    |\n",
+    "| supl                  | [ ]    | [X]    |\n",
+    "| swden                 | [ ]    | [X]    |\n",
+    "| swdir                 | [ ]    | [X]    |\n",
+    "| swdir2                | [ ]    | [X]    |\n",
+    "| swr1                  | [ ]    | [X]    |\n",
+    "| swr2                  | [ ]    | [X]    |\n",
+    "```\n",
+    "\n",
+    "\n",
```

Comment:
It's not clear to me which mode delivers which kind of data. 
Is there a way to point towards the documention of `ndbc_api` or directly NDBC?

Reply:
I couldn't find any, I tried to make it clear but I see how it could be not clear for all users. Do you have any suggestions of the best approach to show this?

Reply:
I think we can make link with the NDBC website. 
I found this: https://www.ndbc.noaa.gov/faq/measdes.shtml that can be put in the description.

Reply:
Sure